dcicutils/s3_utils.py

```diff
@@ -29,6 +29,8 @@ class s3Utils(object):  # NOQA - This class name violates style rules, but a lot
     OUTFILE_BUCKET_TEMPLATE = "elasticbeanstalk-%s-wfoutput"
     RAW_BUCKET_TEMPLATE = "elasticbeanstalk-%s-files"
     BLOB_BUCKET_TEMPLATE = "elasticbeanstalk-%s-blobs"
+    METADATA_BUCKET_TEMPLATE = "elasticbeanstalk-%s-metadata-bundles"
+    TIBANNA_LOGS_BUCKET_TEMPLATE = "elasticbeanstalk-%s-tibanna-logs"
```

Comment:
I would not add this as it isn't correct in the old account (hardcoded to `tibanna-logs`) or the new account.

Reply:
It should default to what works in the legacy account. It doesn't have to work for alpha. So I think you're saying I should just have it be "tibanna-logs".